src/obs-progress.cpp

```diff
@@ -71,14 +72,12 @@ void startTimer()
 
 void stopTimer()
 {
-	try
+	if (_timer->isActive())
 	{
-		//_timer->stop();
-	}
-	catch (...)
-	{
-		// Empty
+		_timer->stop();
```

Comment:
I surrounded the timer stop with an `isActive` check to avoid the Qt exception. Now I back to an OBS shutdown exception.

```
obs.dll!os_atomic_dec_long+0x10
obs.dll!obs_ref_release+0x2b
obs.dll!obs_source_release+0x63
obs.dll!obs_transition_clear+0x189
obs.dll!obs_source_destroy+0x5a
obs.dll!obs_free_data+0x77
obs.dll!obs_shutdown+0x465
obs64.exe!OBSApp::~OBSApp+0xa1
obs64.exe!run_program+0x86e
obs64.exe!main+0x6b8
obs64.exe!WinMain+0xed
obs64.exe!invoke_main+0x32
obs64.exe!__scrt_common_main_seh+0x12e
obs64.exe!__scrt_common_main+0xe
obs64.exe!WinMainCRTStartup+0x9
kernel32.dll!0x7ff9a20b6fd4
ntdll.dll!0x7ff9a2cdcec1
```